lib/FieldType/Mapper/RelationListFormMapper.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace EzSystems\RepositoryForms\FieldType\Mapper;
+
+use eZ\Publish\API\Repository\Repository;
+use eZ\Publish\Core\FieldType\RelationList\Type;
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+use EzSystems\RepositoryForms\Data\FieldDefinitionData;
+use EzSystems\RepositoryForms\FieldType\FieldTypeFormMapperInterface;
+use Symfony\Component\Form\FormInterface;
+
+
+class RelationListFormMapper implements FieldTypeFormMapperInterface
+{
+    /**
+     * @var Repository The API Repository, used to fetch list of available content types
+     */
+    protected $repository;
+
+    /**
+     * @var ConfigResolverInterface Config resolver, for reading current language
+     */
+    protected $configResolver;
+
+    /**
+     * @param Repository $repository
+     * @param ConfigResolverInterface $configResolver
+     */
+    public function __construct(Repository $repository, ConfigResolverInterface $configResolver)
+    {
+        $this->repository = $repository;
+        $this->configResolver = $configResolver;
+    }
+
+    public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, FieldDefinitionData $data)
+    {
+        // Fill a hash with all content types and their ids
+        $contentTypeHash = [];
+        $contentTypeService = $this->repository->getContentTypeService();
+        foreach ($contentTypeService->loadContentTypeGroups() as $contentTypeGroup) {
+            foreach ($contentTypeService->loadContentTypes($contentTypeGroup) as $contentType) {
+                $contentTypeHash[$contentType->identifier] = $contentType->getName(
+                    $this->configResolver->getParameter("languages")[0]
```

Comment:
Use translation helper instead. It's the service behind `ez_content_name()` Twig helper.